src/components/swap/SwapBuyFiatButton.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from 'test-utils'
+
+import SwapBuyFiatButton from './SwapBuyFiatButton'
+
+// TODO(lynnshaoyu): add detailed unit tests for critical flows of component.
```

Comment:
i would recommend doing this before the launch :O

Reply:
+1 we should be adding unit tests for critical flows; adding post-launch defeats the purpose of helping increase our confidence in the code and functionality.

Reply:
sounds good does it have to go in with this PR or a follow up PR is ok?

Reply:
It makes more sense to include here, as unit tests for new code should be included as part of a PR. It will also be easier to follow and understand if we wrote tests, and what tests were written, if there's an issue with the PR.

Reply:
makes sense, adding now!

Reply:
As a general practice @lynnshaoyu, can you convert this into JIRA ticket and reference here as `TODO(WEB-XXX): description` so that we can track it later?